lebedigital/calibration/utils.py

```diff
@@ -0,0 +1,161 @@
+## -- TUM PKM --- atul.agrawal@tum.de--- ##
+import matplotlib.pyplot as plt
+import rdflib
+import pandas as pd
+import numpy as np
+import os
+from pathlib import Path
+import sys
+import fenics_concrete
+
+baseDir1 = Path(__file__).resolve().parents[1]
+sys.path.append(os.path.join(os.path.join(baseDir1, "knowledgeGraph"), "emodul"))
+sys.path.append(os.path.join(baseDir1, "Data"))
```

Comment:
I dont want to tamper with this. I dont remember paths are used where, plus this is cosmetic change.

Reply:
I am 90% sure my suggestion works.
It should not take more then 20 sec to try this. Change the line and see if it works. Then either keep it or go back to last implementation.

> this is cosmetic change

 yes, but unless you are planning on going over your code to make it pretty, its good to have the code as readable as possible when merging. therre is a good chance that the paths will change during the development of the project and this will make it simpler to fix this. so its more then just cosmetic, but also practical.
writing this respone took more time then you would need to just quickly check if my suggestion produces errors ;)

Reply:
changed.